### PR TITLE
doc: more explaination on vnics and security groups

### PR DESCRIPTION
**Pull request by outscale-jju**
Signed-off-by: Jerome Jutteau <jerome.jutteau@outscale.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,61 @@ network and traffic is filtered by (EC2/Openstack-like) security groups.
 Security groups can be applied to any VM interface, and contain a list of simple
 network rules (dropping traffic by default).
 
+
+# Virtual NICs
+
+A Virtual NIC (or vnic) in butterfly allows you to add a virtual network
+interface to your Qemu VM through vhost-user interface. Each vnic are placed
+on a 24 bits network id (vni).
```
Comment (outscale-mgo) [inline README.md]: Each nick have a 24 bits network id call VNI

File: README.md [hunk 2]
```diff
@@ -8,6 +8,61 @@ network and traffic is filtered by (EC2/Openstack-like) security groups.
...
+If two vnics with the same vni are located on different physical hosts,
```
Comment (outscale-mgo) [inline README.md]: s/vni/VNI/

File: README.md [hunk 3]
```diff
@@ -8,6 +8,61 @@ network and traffic is filtered by (EC2/Openstack-like) security groups.
...
+Butterfly will encapsulate VM packets using VXLAN and send it to the
```
Comment (outscale-mgo) [inline README.md]: s/using/over/

File: README.md [hunk 4]
```diff
@@ -8,6 +8,61 @@ network and traffic is filtered by (EC2/Openstack-like) security groups.
...
+corresponding physical host.
+Once received, the packet will be decapsulated and routed to it's final
+destination.
```
Comment (outscale-mgo) [inline README.md]: http://0.tqn.com/d/horror/1/L/n/L/2/-/The-Final-Destination.jpg

File: README.md [hunk 5]
```diff
@@ -8,6 +8,61 @@ network and traffic is filtered by (EC2/Openstack-like) security groups.
...
+Each vnic created with the same vnic are located on the same network.
```
Comment (outscale-mgo) [inline README.md]: s/vnic/VNI/

File: README.md [hunk 6]
```diff
@@ -8,6 +8,61 @@ network and traffic is filtered by (EC2/Openstack-like) security groups.
...
+and low latency communication between VM.
+
+e.g. create of a new vnic "vnic_1" on vni "1337":
+```
+butterfly nic add --ip 42.0.0.1 --mac 52:54:00:12:34:01 --vni 1337 --id vnic_1
+```
+
+# Filtering
+
+Vnic traffic is filtered using an integrated firewall inside Butterfly
```
Comment (outscale-mgo) [inline README.md]: s/Vnic/VMs/

**Comment by outscale-jju** [comment]
merged

**Pull request closed without merging**